gradle/solr/packaging.gradle

```diff
@@ -60,7 +60,11 @@ configure(allprojects.findAll {project -> project.path.startsWith(":solr:contrib
     // An aggregate that configures lib and test-lib in a temporary location.
     task assemblePackaging(type: Sync) {
       from "README.md"
-      
+
+      from "manifest.json"
+
+      from "src/resources/solr-default-tika-config.xml"
```

Comment:
You can put this in the `extraction` module's build.gradle using

```
assemblePackaging {
  from "src/resources/solr-default-tika-config.xml"
}
```

Reply:
I removed this. The `solr-default-tika-config.xml` is already included in the JAR, given that it's in `src/resources`